onnxruntime/test/framework/tunable_op_test.cc

```diff
@@ -263,8 +263,7 @@ TEST(TunableOp, OpWrapsMutableFunctor) {
 
 class VecAddMoveOnlyFunctor {
  public:
-  VecAddMoveOnlyFunctor() {
-  }
+  VecAddMoveOnlyFunctor() = default;
```

Comment:
Any background why this change is made? Just for simplifying the code?

Reply:
Recommended by [Dmitri](https://github.com/yuslepukhin)